include/uapi/sound/sof/tokens.h

```diff
@@ -135,6 +135,7 @@
 
 /* Processing Components */
 #define SOF_TKN_PROCESS_TYPE                    900
+#define SOF_TKN_PROCESS_BASE_CONFIG_EXT	  901
```

Comment:
I think this is absolutely terrible. This should not be part of topology.

Reply:
@plbossart do you have a better suggestion of where it shoudl come from? We cannot use the manifest because existing modules would not have this info in the manifest. And we've been given clear marching orders that not supporting these existing modules is not an option.

Reply:
@ranj063 Marching orders from whom? This is broken, period. This is a firmware issue, not ours to fix with bungees and duct tape.

Reply:
I could accept that this is a temporary solution, but that looks like a long-term way of defining the kernel-firmware ABI with a topology token, and that's unacceptable. There's got to a better solution so that this temporary solution is removed at some point.

Reply:
> The other option is to have this information for new, SOF modules in manifest (since we can add it), but we still need something for the ones that we don't control.

I would be ok with that approach. I just don't want to use a work-around as a long-term solution.

Reply:
and btw we still need means to check that the topology only uses modules that are available. Currently this leads to IPC errors which is quite bad.